.github/workflows/interop-tests.yml

```diff
@@ -5,9 +5,7 @@
 # to that directory
 name: Interoperability tests with GnuTLS and NSS
 on:
-  schedule:
-    - cron: '55 02 * * *'
-  workflow_dispatch:
+  pull_request:
```

Comment:
This test takes about 40 minutes to run, I'm not sure we want to convert this nightly test to a per-pull-request action

Reply:
This is in the DROP commit, so won't be in the final version